src/config.hpp

```diff
@@ -15,11 +16,13 @@ using color = const uint32_t;
 /// Actual window width will be equal to dxp_width + (dxp_padding * 2)
 ///
 const uint dxp_width = 0;
-const uint dxp_height = 150;
+const uint dxp_height = 250;
 
 const int16_t dxp_x = 0; ///< X coordinate of window's top left corner
 const int16_t dxp_y = 0; ///< Y coordinate of window's top left corner
 
+const bool centering = false;///< Place dxp window centered on screen
+
```

Comment:
Whats this

Reply:
leftovers of my centering realization. will delete